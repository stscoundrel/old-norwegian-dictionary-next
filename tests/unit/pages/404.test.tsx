import ReactDOM from 'react-dom'
import Page404 from 'pages/404'
import renderer from 'react-test-renderer'

describe('404 page page', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Page404/>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<Page404 />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
