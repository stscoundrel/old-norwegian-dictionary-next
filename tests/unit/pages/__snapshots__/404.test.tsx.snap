// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page page Matches snapshot 1`] = `
Array [
  <main
    className="container"
  >
    <section
      className="section"
    >
      <h1>
        Page not found
      </h1>
      <p>
        Huh, that's weird.
      </p>
    </section>
  </main>,
  <footer
    className="section"
  >
    <div
      className="container"
    >
      <p>
        Copyright Â© 
        2021
      </p>
    </div>
  </footer>,
]
`;
